src/main/java/application/FormController.java

```diff
@@ -102,6 +111,7 @@ public SubmitResponseBean submitForm(@RequestBody SubmitRequestBean submitReques
         FormRecordProcessorHelper.processXML(formEntrySession.getSandbox(), formEntrySession.submitGetXml());
         SubmitResponseBean submitResponseBean = new SubmitResponseBean(formEntrySession);
         log.info("Submit response bean: " + submitResponseBean);
+        sessionRepo.delete(submitRequestBean.getSessionId());
```

Comment:
This is a form entry session, so once the form's submitted we don't need this information anymore. We could keep it around and expire it somehow, but I can't think of a situation when we'd to restore a finished session. Can you @benrudolph